docs/setup/independent/install-kubeadm.md

```diff
@@ -87,8 +88,8 @@ You can use the following commands to install Docker on your system:
 Install Docker from Ubuntu's repositories:
 
 ```bash
-apt-get update
-apt-get install -y docker.io
+sudo apt-get update
```

Comment:
It's mentioned that above that being the root user here is assumed

Reply:
I am looking carefully again and I do not see this instruction.

We are also wondering why, later on, they need to become root and can't use `sudo`. https://kubernetes.io/docs/setup/independent/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl cc: @caseydavenport

Reply:
ok, seems like it had moved down a couple of sentences... let's move it up here for clarity:

lookup this in the doc and move up here please:

> Please proceed with executing the following commands based on your OS as root. You may become the root user by executing sudo -i after SSH-ing to each host.

Reply:
I prefer to not repeat sudo everywhere in docs